### DISCUSSION TITLE
Radial Forces

### DISCUSSION THREAD

**Discussion by hserdogan94** [petrobras/ross#1085] [Q&A]
Hello, 

I am studying rotor dynamics for my master's thesis. I should represent axial and radial forces on the ball bearings in my modeling. For axial forces, I think I can represent them on the nodes of the bearing corresponding over the shaft. But for the radial forces, do you know how I can model them? I would be very happy if someone could help me. 

Thanks in advance.

**Comment by jguarato** [comment]
Hi @hserdogan94,

To model the forces on the ball bearings, you should create an array of forces with a size equal to the total degrees of freedom (`rotor.ndof`) in your model:

    F = np.zeros(rotor.ndof)

Both axial and radial forces should be applied to the nodes where the ball bearings are located.

In the context of ROSS, radial forces are decomposed into the x and y degrees of freedom:

    dof_x = node * rotor.number_dof + 0
    F[dof_x] = force_x_value

    dof_y = node * rotor.number_dof + 1
    F[dof_y] = force_y_value

Axial forces are applied in the z direction (only in 6 DoF models):

    dof_axial = node * rotor.number_dof + 2
    F[dof_axial] = axial_force_value

Hope this helps, and best of luck with your thesis!
